packages/engine/src/scene/functions/loadGLTFModel.ts

```diff
@@ -93,6 +94,11 @@ export const parseGLTFModel = (
 ) => {
   // console.log(sceneLoader, entity, component, sceneProperty, scene)
 
+  // Enable scene layer
+  scene.traverse((obj: Object3D) => {
+    obj.layers.enable(CameraLayers.Scene)
+  })
```

Comment:
this might also be a good target for a unit test for `parseGLTFModel`

e.g. `it('should enable scene layer for camera')`

Reply:
good idea! i fixed the loadGLTFModel test and made it now check for this. will need to extend this in the future, but this particular issue will be caught now.